test/Test/Data/List.purs

```diff
@@ -226,6 +226,11 @@ testList = do
   assert $ (take 2 (l [1, 2, 3])) == l [1, 2]
   assert $ (take 1 nil) == nil
 
+  log "take should keep the specified number of items from the end of an list, discarding the rest"
```

Comment:
take**End** should keep the specified number of items from the end of an list, discarding the rest